packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -67,6 +67,10 @@ class DescopeWc extends BaseDescopeWc {
     BaseDescopeWc.sdkConfigOverrides = config;
   }
 
+  static get sdkConfigOverrides() {
```

Comment:
There is a test in one of the frameworks that verifies the sdkConfigOverrides is updated 
However, since we do not have a getter, the test cannot retrieve the updated value 
I’m not sure how it was working in the first place